apps/frontend/src/app/5_pages/BitocracyPage/components/ProposalTitle/ProposalTitle.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FC, useMemo } from 'react';
+
+import { Tooltip, TooltipTrigger } from '@sovryn/ui';
+
+import { ProposalProps } from '../../BitocracyPage.types';
+
+const titleCharacterLimit = 50;
```

Comment:
Might be good to move this to `apps/frontend/src/app/5_pages/BitocracyPage/BitocracyPage.types.ts` since I think it will be used for the New Proposal functionality (but check with the other devs and disregard if irrelevant)

Reply:
This won't be used anywhere else as the limit for titles in new proposals is 140.

Reply:
gotcha thanks